.eslintrc.js

```diff
@@ -121,6 +121,16 @@ module.exports = {
         jest: true
       }
     },
+    {
+      // Matches Puppeteer environment in jest.config.mjs
+      // to ignore unknown Jest Puppeteer globals
+      files: ['**/*.puppeteer.test.{js,mjs}'],
```

Comment:
Moved from [**packages/govuk-frontend/.eslintrc.js**](https://github.com/alphagov/govuk-frontend/pull/4205/files#diff-5522c69942e4ae29ac022153cddf7837010848662f1ad2afd987ea9d6643bbbf) to allow new Puppeteer tests in the Review app to use `page` globals etc